src/Sylius/Component/Core/Model/ChannelPricing.php

```diff
@@ -24,7 +22,7 @@ class ChannelPricing implements ChannelPricingInterface
     protected $id;
 
     /**
-     * @var ChannelInterface
+     * @var string
      */
     protected $channel;
```

Comment:
I was thinking about `$channelCode` also, but I wanted to be consistent with convention from translations (as they're similar concept) - there is `$locale` there, not `$localeCode`. I doesn't make any difference for me, but I want to hear your (and anyone else) opinion.